### PR TITLE
[New] Add leaflet-groupedlayercontrol@0.6.0 w/ git auto-update

### PR DESCRIPTION
**Pull request by sufuf3**
Pull request for issue: #10674
Related issue(s): #10709 #11170

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/ismyrnow/leaflet-groupedlayercontrol
 * Official website (optional, not the repository): 
 * NPM package url (optional): https://www.npmjs.com/package/leaflet-groupedlayercontrol
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 14
   - Count of stars: 108
   - Count of forks: 60
 * NPM download stats (optional):
   - Downloads in the last day: 1
   - Downloads in the last week: 34
   - Downloads in the last month: 235

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
@cdnjs/intern3
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@extend1994 Thank you for your suggestion.
I modify it, please help me review this PR, thanks.

**Comment by sufuf3** [comment]
ping @hare1039 @clairetsai818

**Comment by sufuf3** [comment]
@dakshshah96 I added the `homepage` field.
Please help me review this PR again, Thank you.

**Comment by sufuf3** [comment]
@cdnjs/library-reviewer 
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@hare1039 @qwe19272375 @pvnr0082t @maruilian11 
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@hare1039 @qwe19272375 @pvnr0082t @maruilian11
Please help me review this PR, Thank you.

File: ajax/libs/leaflet-groupedlayercontrol/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "leaflet-groupedlayercontrol",
+  "version": "0.6.0",
+  "description": "A Leaflet layer control with support for grouping overlays together",
+  "filename": "leaflet.groupedlayercontrol.min.js",
+  "repository": {
```
Comment (qwe19272375) [inline ajax/libs/leaflet-groupedlayercontrol/package.json]: please change repository url to `https://github.com/ismyrnow/leaflet-groupedlayercontrol`

**Comment by sufuf3** [comment]
@qwe19272375 I modified the url of repository
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@qwe19272375 Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@pvnr0082t @maruilian11 
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@maruilian11 @pvnr0082t
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@maruilian11 The commits relationship between `[New] Manually add timbre v13.03.01 ~ v14.11.25`  and `Add leaflet-groupedlayercontrol@0.6.0 w/ git auto-update` is?

Sorry that I am a bit confused.

**Comment by maruilian11** [comment]
Just a rebase fault.

**Comment by sufuf3** [comment]
Thank you.

**Pull request merged**